lib/paper_trail/queries/versions/where_attribute_changes.rb

```diff
@@ -23,12 +23,16 @@ def execute
               @version_model_class, @attribute
             )
           end
-
-          case @version_model_class.columns_hash["object_changes"].type
+          column_type = @version_model_class.columns_hash["object_changes"].type
+          case column_type
           when :jsonb, :json
             json
           else
-            text
+            raise UnsupportedColumnType.new(
+              method: "where_attribute_changes",
+              expected: "json/b",
```

Comment:
Is this a canonical way to say either json or jsonb? I looked quickly through the Postgres docs, just as an example, and couldn't find any reference to it, but I don't know if we refer to that combination elsewhere in this repo in that way.

I think being explicit here may help readers of the exception's message:

```suggestion
              expected: "json or jsonb",
```